app/views/incomes/bonusPaymentAmount.scala.html

```diff
@@ -51,6 +51,9 @@
     if(bonusPeriod == "year"){
         Messages("tai.bonusPaymentsAmount.year.message")
     }
+    else{
```

Comment:
Do we need this else condition?

Reply:
This is to resolve a bug where if bonus period is not equal year the input field would try to toString  the hint variable and returns empty parenthesis for some reason.